### PR TITLE
LCAM-286|Add Entity and Repository classes for Contributions

### PR DESCRIPTION
**Pull request by venkat980**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-286)

Add Entity and Repository classes for Contributions.

## Checklist

Before you ask people to review this PR:

- [x] Tests should be passing: `./gradlew test`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by muthusubra — COMMENTED** [review]
What about Impl and Service classes?

**Comment by venkat980** [comment]
> What about Impl and Service classes?

it was decided that we will only create the Entity and Repository classes for now, same as LCAM-287.

File: maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionsEntity.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    @Column(name = "CONT_FILE_ID")
+    private Integer contributionFileId;
+
+    @Column(name = "EFFECTIVE_DATE", nullable = false)
+    private LocalDate effectiveDate;
+
+    @Column(name = "CALC_DATE", nullable = false)
+    private LocalDate calcDate;
+
+    @Column(name = "CONTRIBUTION_CAP", nullable = false)
```
Comment (Mhowell494) [inline maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionsEntity.java:40]: This looks like a monetary value, so I think BigDecimal is more appropriate.

File: maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionsEntity.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    private Integer contributionCap;
+
+    @Column(name = "MONTHLY_CONTRIBS", nullable = false)
+    private Integer monthlyContributions;
```
Comment (Mhowell494) [inline maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionsEntity.java:44]: This looks like a monetary value, so I think BigDecimal is more appropriate.

File: maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionsEntity.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+    @Column(name = "UPFRONT_CONTRIBS")
+    private Integer upfrontContributions;
```
Comment (Mhowell494) [inline maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionsEntity.java:47]: This looks like a monetary value, so I think BigDecimal is more appropriate.

**Review by Mhowell494 — CHANGES_REQUESTED** [review]

**Review by Mhowell494 — APPROVED** [review]

**Pull request merged**